src/removeTasks.js

```diff
@@ -0,0 +1,11 @@
+// eslint-disable-next-line import/no-cycle
+import { ul, tasks } from './index.js';
+
+export default function removeTasks(event) {
+  event.stopPropagation();
+  const li = event.target.parentNode;
+  const index = Array.prototype.indexOf.call(ul.children, li);
+  tasks.splice(index, 1);
+  li.remove();
+  localStorage.setItem('tasks', JSON.stringify(tasks));
+}
```

Comment:
- [ ] Just as the previous reviewer pointed out, when a task is deleted, the remaining task indexes should get updated and be ordered from 1...3. Currently when I added 5 tasks and deleted tasks 3 and 4, the remaining tasks index does not get updated in an orderly manner as expected. Kindly work on that. 👍🏽
- [ ] Let's try to not have Linter checks disabled in our codebase. It is there to help us write better clean and more professional code. Kindly fix the error 👍🏽

<img width="414" alt="Screenshot 2023-03-31 at 12 42 12 AM" src="https://user-images.githubusercontent.com/50941074/228995251-23b702e7-eee1-4e20-a106-1617277acead.png">

Reply:
Hello @AmaduKamara,

Could you please guide me through the solution?. I'm really lost on how to solve the index requirement :/